tests/foreman/ui/test_rhcloud_inventory.py

```diff
@@ -16,36 +16,42 @@
 
 :Upstream: No
 """
+from datetime import datetime
+from datetime import timedelta
+
 import pytest
+from nailgun import entities
 
+from robottelo.api.utils import wait_for_tasks
 from robottelo.rh_cloud_utils import get_local_file_data
 from robottelo.rh_cloud_utils import get_remote_report_checksum
+from robottelo.rh_cloud_utils import get_report_data
 
 
-@pytest.mark.tier3
-def test_rhcloud_inventory_e2e(organization_ak_setup, registered_hosts, session):
-    """Generate report and verify its basic properties
+def setting_update(name, value):
+    """
```

Comment:
we have settings update fixture in `api_fixtures.py` can't we use that one.

Reply:
@devendra104 I was thinking of using that but that fixture doesn't seem useful for my test scenarios. I am using `setting_update` in between tests and not just as pre-setup.
There are two ways to enable/disable inventory settings, from inventory page and other option is using main settings.
I want to test both of them and using fixture you mentioned won't help in that case.
I have added `inventory_settings` to make sure those settings are disabled at start and end of the test.
In test I will first test setting present on inventory page and then I will test options from main settings as well and
to do that I will use `setting_update`.